src/media-gallery/media-gallery.js

```diff
@@ -92,12 +96,20 @@ component.bindings = {
     allowEnlarge: '<?',
     onEnlarge: '&?',
     limitTo: '<?',
-    onViewAll: '&?'
+    onViewAll: '&?',
+    enableListView: '<?',
+    onSetView: '&?'
 };
 
 component.template = `
+<div ng-if="vm.enableListView && (vm.media && vm.media.length > 0)" class="button-container right row media-gallery__view-controls">
+    <ul class="button-group row">
+        <li><a class="btn svgicon svgicon--icon_grid-view media-gallery__btn" ng-class="{'media-gallery__btn--active': vm.view === vm.viewPreferences.TILE}" ng-click="vm.setView(vm.viewPreferences.TILE)"></a></li>
```

Comment:
Try to use <button>s here if you can for semantics and accessibility

Reply:
we have a button class for svgicons now: https://buildium-pattern-library.herokuapp.com/section-2.html#kssref-2-8-2

you should include text inside both links and buttons (you can hide it w/ the screen reader class) that describes the action. Empty links/buttons I believe are considered invalid

Reply:
Not sure I understand 'use s', could you explain further?

Reply:
The UX designed for this feature doesn't match those button classes. Thoughts?

Reply:
![image](https://user-images.githubusercontent.com/41081434/43468788-d24745ce-94b2-11e8-934c-5c93b3d33049.png)

Reply:
Oh this wouldn't change the styling hopefully?

Reply:
theoretically shouldn't change the style :)